netlink-packet-route/src/rtnl/route/nlas/next_hops.rs

```diff
@@ -28,6 +27,7 @@ buffer!(NextHopBuffer {
     flags: (u8, 2),
     hops: (u8, 3),
     interface_id: (u32, 4..8),
+    // This might contain other NLAs too
     gateway_nla: (slice, GATEWAY_OFFSET..),
```

Comment:
Maybe this should be renamed `nlas` since we can have different sorts of attributes.

Reply:
Sure, done.